docker-compose.yml

```diff
@@ -208,16 +209,27 @@ services:
      - 5430:5432
 
   metabase:
-    image: metabase/metabase:${METABASE_IMAGE_TAG}
+    image: bahmni/bahmni-metabase
     restart: on-failure:[3]
     profiles: ["metabase","default"]
     environment:
+      MB_HOST: ${METABASE_HOST:?}
+      MB_PORT: ${METABASE_PORT:?}
+      MB_ADMIN_EMAIL: ${METABASE_ADMIN_EMAIL:?}
+      MB_ADMIN_FIRST_NAME: ${METABASE_ADMIN_FIRST_NAME:?}
+      MB_ADMIN_LAST_NAME: ${METABASE_ADMIN_LAST_NAME:?}
+      MB_ADMIN_PASSWORD: ${METABASE_ADMIN_PASSWORD:?}
       MB_DB_TYPE: postgres
       MB_DB_DBNAME: ${METABASE_DB_NAME:?}
       MB_DB_PORT:  ${METABASE_DB_PORT:?}
       MB_DB_USER: ${METABASE_DB_USER:?}
       MB_DB_PASS: ${METABASE_DB_PASSWORD:?}
       MB_DB_HOST: ${METABASE_DB_HOST:?}
+      OPENMRS_DB_HOST: ${OPENMRS_DB_HOST:?}
+      OPENMRS_DB_NAME: ${OPENMRS_DB_NAME:?}
+      OPENMRS_DB_USERNAME: ${OPENMRS_DB_USERNAME:?}
+      OPENMRS_DB_PASSWORD: ${OPENMRS_DB_PASSWORD:?}  
+      MB_MYSQL_ENGINE: ${METABASE_MYSQL_ENGINE:?}
```

Comment:
Does this variable refer the type of OpenMRS DB ? If yes then we should consider giving a better name . Probably `OPENMRS_DB_TYPE`

Reply:
Hi Mohan , We have updated the METABASE_MYSQL_ENGINE to  OPENMRS_DB_TYPE and checked in the code